README.md

```diff
@@ -2,23 +2,23 @@
 
 This repository contains the samples that demonstrate the functionalities of WinUI chart controls.
 
-Instead of SfChart, we have provided the following chart types in WinUI.
+We have provided the following chart types in WinUI.
 
-* Cartesian chart
-* Circular chart
+* Cartesian charts
+* Circular charts
 * Pyramid chart
 * Funnel chart
-* Polar chart
+* Polar charts
 
-#### Cartesian chart
+#### Cartesian charts
 
-A Cartesian chart is an elegant way to visualize data. Charts supports more than 15 Cartesian chart types such as line, spline, column, bar, area, scatter, and bubble.
+A Cartesian chart is an elegant way to visualize data. Charts supports more than 15 Cartesian chart types such as line, spline, column, area, scatter, and bubble.
```

Comment:
The Cartesian chart is an elegant way to visualize data. It supports more than 15 Cartesian chart types such as line, spline, column, bar, area, scatter, bubble, etc.

Reply:
changed.